docs/cardinal/game/query.mdx

```diff
@@ -56,7 +64,7 @@ You can easily create a new query and register it to the world by following thes
 
             // Register queries
             // NOTE: You must register your queries here for it to be accessible.
-            err := cardinal.RegisterQuery[query.WorldVarsRequest, query.WorldVarsResponse](w, "world-vars", query.WorldVars))
+            err := cardinal.RegisterQuery[query.PlayerHealthRequest, query.PlayerHealthResponse](w, "world-vars", query.PlayerHealthResponse))
```

Comment:
The third parameter shouldn't be `query.PlayerHealthResponse`, it should correspond to the query function

Reply:
whoops, missed this.